langchain/document_loaders/parsers/audio.py

```diff
@@ -12,9 +12,18 @@ class OpenAIWhisperParser(BaseBlobParser):
     def lazy_parse(self, blob: Blob) -> Iterator[Document]:
         """Lazily parse the blob."""
 
+        import io 
         import openai
 
+        # Need if we pass an in-memory blob from AudioSplitter().lazy_split(video)
+        file_obj = io.BytesIO(blob.data)
+        file_obj.name = "input_split.mp3"
```

Comment:
Is this name important for anything in terms of how the openai API uses it? Is it OK that multiple audio files share the same filename?